packages/transitive-overlay/src/index.tsx

```diff
@@ -269,7 +278,54 @@ const TransitiveCanvasOverlay = ({
         }}
         type="line"
       />
-
+      {showRouteArrows && (
+        <Layer
+          id="route-arrows-walk"
+          filter={[
+            "all",
+            ["==", "type", "street-edge"],
+            ["==", "mode", "WALK"]
+          ]}
+          type="symbol"
+          layout={{
+            "symbol-placement": "line",
+            "icon-image": "arrow-icon",
+            "icon-size": 0.1,
+            "symbol-spacing": 10,
+            "icon-allow-overlap": true,
+            "icon-ignore-placement": true,
+            "icon-offset": [0, 8000]
+          }}
+          paint={{
+            "icon-color": defaultColorOverrideArrow,
+            "icon-opacity": 0.8
+          }}
+        />
+      )}
+      {showRouteArrows && (
+        <Layer
+          id="route-arrows"
+          type="symbol"
+          filter={[
+            "all",
+            ["!=", "type", "street-edge"],
+            ["!=", "mode", "WALK"]
+          ]}
+          layout={{
+            "symbol-placement": "line",
+            "icon-image": "arrow-icon",
+            "icon-size": 0.1,
+            "symbol-spacing": 10,
+            "icon-allow-overlap": true,
+            "icon-ignore-placement": true,
+            "icon-offset": [0, 8000]
+          }}
+          paint={{
+            "icon-color": ["get", "color"],
```

Comment:
Doesn't this work out to always be `routeColorOverride`? Why not just use that here and then you don't need `routeColorOverrideArrow`?

Reply:
We want the routeColorOverrideArrow color to be there if there is no color to pull from: a 'default' in case there is no color to match up.

Reply:
Hmm... Should we use the `["get", "color"]` instead then? Is there a use case for the walk arrows being a different color than the other arrows?

Reply:
There is no use case and we've just agreed to drop the teal thing until it comes up later. Expect a new push soon! :)

Reply:
If it even comes up later.